server/handler.go

```diff
@@ -418,32 +418,56 @@ func (h *Handler) GetHistory(start time.Time) ([]operator.OpHistory, error) {
 }
 
 // SetAllStoresLimit is used to set limit of all stores.
-func (h *Handler) SetAllStoresLimit(rate float64, limitType storelimit.Type) error {
-	c, err := h.GetOperatorController()
-	if err != nil {
-		return err
+func (h *Handler) SetAllStoresLimit(ratePerMin float64, limitType storelimit.Type) error {
+	cfg := h.GetScheduleConfig().Clone()
+	switch limitType {
+	case storelimit.AddPeer:
+		for storeID := range cfg.StoreLimit {
+			sc := config.StoreLimitConfig{
+				AddPeer:    ratePerMin,
+				RemovePeer: cfg.StoreLimit[storeID].RemovePeer,
+			}
+			cfg.StoreLimit[storeID] = sc
+		}
+	case storelimit.RemovePeer:
+		for storeID := range cfg.StoreLimit {
+			sc := config.StoreLimitConfig{
+				AddPeer:    cfg.StoreLimit[storeID].AddPeer,
+				RemovePeer: ratePerMin,
+			}
+			cfg.StoreLimit[storeID] = sc
+		}
 	}
```

Comment:
```suggestion
for storeID := range cfg.StoreLimit {
	cfg.StoreLimit[storeID][limitType] = ratePerMin 
}
```

Reply:
Are you worried that there may be `type` not initialized?

Reply:
`StoreLimit[storeID]` is a struct not a map.